### ISSUE TITLE
Unable to load required framework: ext@null

### ISSUE THREAD

**Issue by JernejHabjan** [sencha/ext-angular#185]
Hi everyone!

after following instructions in README.md I successfully created app Cool Ext Angular App, but it failed to start, because ext dependency might not have installed with npm install?

```cmd
npm start

> @sencha/ext-angular-boilerplate@6.7.0 start C:\Users\jerne\Desktop\NEWEST\cool-ext-angular-app
> npm run dev

> @sencha/ext-angular-boilerplate@6.7.0 dev C:\Users\jerne\Desktop\NEWEST\cool-ext-angular-app
> webpack-dev-server --env.verbose=no --env.environment=development  --env.browser=true  --env.watch=yes

undefined
i [ext]: ext-webpack-plugin v6.7.0, Ext JS v6.7.0 Commercial Edition, Sencha Cmd v6.7.0.36, webpack v4.29.0, angular v7.2.2
i [ext]: Building for development
i [ext]: genProdData: false
i ｢wds｣: Project is running at http://0.0.0.0:1962/
i ｢wds｣: webpack output is served from /
i ｢wds｣: Content not from webpack is served from ./build
i ｢wds｣: 404s will fallback to /index.html
i [ext]: Adding ext-angular\ext.js and ext-angular\ext.css to index.html
i [ext]: Building Ext bundle at: \build\ext-angular
i [ext]: [ERR] Directory \build\ext-angular\node_modules\@sencha\ext not recognized as a framework
i [ext]: [ERR] Unable to load required framework: ext@null
here
false
i [ext]: Opening browser at http://localhost:1962
i [ext]: Completed ext-webpack-plugin processing
× ｢wdm｣: Built at: 01/25/2019 11:31:58 AM
Entrypoint polyfills = polyfills.js polyfills.js.map
Entrypoint main = main.js main.js.map

ERROR in i [ext]: Directory \build\ext-angular\node_modules\@sencha\ext not recognized as a framework

ERROR in i [ext]: Unable to load required framework: ext@null

ERROR in 321
i ｢wdm｣: Failed to compile.
```

**Comment by Romanitch** [comment]
Same issue ....
i [ext]: ext-webpack-plugin v6.7.0, Ext JS v6.7.0 Commercial Edition, Sencha Cmd v6.7.0.36, webpack v4.29.0, angular v7.2.2
i [ext]: Building for development
i [ext]: genProdData: false
i ｢wds｣: Project is running at http://0.0.0.0:1962/
i ｢wds｣: webpack output is served from /
i ｢wds｣: Content not from webpack is served from ./build
i ｢wds｣: 404s will fallback to /index.html
i [ext]: Adding ext-angular\ext.js and ext-angular\ext.css to index.html
i [ext]: Building Ext bundle at: \build\ext-angular
i [ext]: [ERR] Directory \build\ext-angular\node_modules\@sencha\ext not recognized as a framework
i [ext]: [ERR] Unable to load required framework: ext@null
false
i [ext]: Opening browser at http://localhost:1962
here
i [ext]: Completed ext-webpack-plugin processing
i ｢wdm｣: Failed to compile.
× ｢wdm｣: Built at: 2019-01-25 16:54:27
Entrypoint polyfills = polyfills.js polyfills.js.map
Entrypoint main = main.js main.js.map

ERROR in i [ext]: Directory \build\ext-angular\node_modules\@sencha\ext not recognized as a framework

ERROR in i [ext]: Unable to load required framework: ext@null

ERROR in 321

**Comment by mgusmano** [comment]
have you tried with 6.7.0?

**Comment by mgusmano** [comment]
Let me know, and if there are still issues we can get a fix in as 6.7.1

**Comment by JernejHabjan** [comment]
Hi everyone

Yes, its working now.
Thanks.
I believe issue can be closed now.
